external/scala/test.properties

```diff
@@ -3,7 +3,4 @@ script="scala-test.sh"
 home_path=""
 tag_version="v2.13.2"
 sbt_version="1.2.8"
-debian_packages="git wget tar curl gpg gpg-agent"
-alpine_packages="bash git wget tar curl gnupg"
-centos_packages="git wget tar curl gpg"
-ubi_packages="git wget tar curl gpg"
+ubuntu_packages="git wget tar curl gpg"
```

Comment:
Same here, the reason change `git wget tar curl gpg gpg-agent` to `git wget tar curl gpg`?